pico8/demos/upsidedown.py

```diff
@@ -286,9 +286,9 @@ def upsidedown_game(g, smallmap=False, flipbuttons=False, flipsounds=False):
             it.__next__()
     except StopIteration:
         pass
-    
 
-def main(orig_args):
+
+def main(orig_args=None):
```

Comment:
The setuptools console_scripts require that entry points be argumentless.  Here, I've added a default so that `main()` works.  A similar change was made to the demo script.